### PR TITLE
Prevent the search to get focus when deleting/pasting an entry

### PR DESCRIPTION
**Pull request by bartsch-dev**
Fixes: #2208

When you delete an entry the focus will no longer jump to the searchbar, instead it will select the next entry (also counts for pasting).

### CODE REVIEW

**Comment by bartsch-dev** [comment]
Now it works for cutting entries too.
I removed the code duplicate, although I'm not sure about the the pattern (not able to use `runCommand(Actions.DELETE)`).

File: src/main/java/net/sf/jabref/gui/BasePanel.java [hunk 1]
```diff
@@ -783,7 +785,10 @@ private void delete() {
         getUndoManager().addEdit(compound);
 
         markBaseChanged();
-        frame.output(formatOutputMessage(Localization.lang("Deleted"), entries.size()));
+        frame.output(formatOutputMessage(!cut ? Localization.lang("Deleted") : Localization.lang("Cut"), entries.size()));
+
```
Comment (Siedlerchr) [inline src/main/java/net/sf/jabref/gui/BasePanel.java]: Either you change the condition and put cut instead of not cut or you wrap it in if and else

File: src/main/java/net/sf/jabref/gui/maintable/MainTableSelectionListener.java [hunk 2]
```diff
@@ -183,11 +183,10 @@ public void editSignalled() {
 
     public void editSignalled(BibEntry entry) {
         final BasePanelMode mode = panel.getMode();
-        EntryEditor editor = panel.getEntryEditor(entry);
```
Comment (Siedlerchr) [inline src/main/java/net/sf/jabref/gui/maintable/MainTableSelectionListener.java]: Why did you change this?

Comment (bartsch-dev) [inline src/main/java/net/sf/jabref/gui/maintable/MainTableSelectionListener.java]: The new editor was only needed if the condition was met, and previously the focus was possibly set to the wrong editor (if the condition wasn't met).

File: src/main/java/net/sf/jabref/gui/BasePanel.java (hunk 1 above)
Comment (bartsch-dev) [inline src/main/java/net/sf/jabref/gui/BasePanel.java]: I changed it to cut (but didn't change the semantic').

**Comment by Siedlerchr** [comment]
To me code looks good :+2:
